apps/agreement/package.json

```diff
@@ -24,7 +24,7 @@
     "apm:publish:patch": "aragon apm publish patch --files public/ --prepublish-script apm:prepublish"
   },
   "dependencies": {
-    "@aragon/os": "5.0.0-beta.1"
+    "@aragon/os": "^5.0.0-beta.2"
```

Comment:
Probably best to keep this pinned so we can make changes consciously :).

Reply:
Cool, I'll unpin it in the next PR I'm about to push :)